<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="newPB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadPB">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>load</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineIme">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ime Ljubimca</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSave">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ime fajla gde se sejvuju novi podaci</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineLoad">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ime fajla odakle da loadujem</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
